src/kbmod/region_search.py

```diff
@@ -14,6 +17,11 @@ def _chunked_data_ids(dataIds, chunk_size=200):
         yield dataIds[i : i + chunk_size]
 
 
+def _trim_uri(uri):
+    """Trim the URI to remove the file:// prefix."""
+    return uri[7:]
```

Comment:
I wasn't sure a great way to standardize it, so I wanted to have a default option that the user could switch out with the `trim_uri_func` parameter. The LSST `ResourcePath` looks helpful, but for now just swapping out the string slicing for urllib since the speed shouldn't matter much here.